Jolt/Core/FPControlWord.h

```diff
@@ -126,6 +126,35 @@ class FPControlWord : public NonCopyable
 	uint32		mPrevState;
 };
 
+#elif defined(JPH_CPU_PPC64)
+
+template <uint64 Value, uint64 Mask>
```

Comment:
Does this code work? I would expect a change in FPException.h and FPFlushDenormals.h too.